### DISCUSSION TITLE
[feature_request] Webhook connectivity without plaintext query string tokens

### DISCUSSION THREAD

**Discussion by LearningMLTranslations** [community/community#58383] [Enterprise]
### Select Topic Area

Product Feedback

### Body

Note: I can't make this request from my enterprise account because of contribution limitations.

I am encountering an issue connecting GitHub webhooks to Splunk via HEC. GitHub requires "allowQueryStringAuth" to be set in the .conf stanza of the installed Splunk app but this is not possible in our environment per Splunk support.

My request is for GitHub webhooks to offer a separate authentication mechanism besides query string auth via raw?token= to pass in the HEC token.

**Comment by VitalyBurshteyn** [comment]
Hi, just curious, did you ever get webooks to work with Splunk HEC?
